packages/shared-utils/src/consts.ts

```diff
@@ -0,0 +1,17 @@
+export const DefaultPermissions = {
+  OPENID: "openid",
+  USERINFO: "userinfo",
+  EMAIL: "email",
+};
+
+export const UiClientPermissions = {
+  READ_KEYS: "read:keys",
+  CREATE_KEYS: "create:keys",
+  DELETE_KEYS: "delete:keys",
+  ROTATE_KEYS: "rotate:keys",
+};
+
+export const WidgetHostPermissions = {
+  READ_WIDGET_ENDPOINTS: "read:widget-endpoints",
+  WRITE_WIDGET_ENDPOINTS: "write:widget-endpoints",
+};
```

Comment:
Enums didn't play well with the webpack loader. Converting these to consts fixed the issue when trying to use the shared package with the ucp-ui.